### ISSUE TITLE
[Question] Support for Anomaly Detection Training

### ISSUE THREAD

**Issue by shaggygi** [dotnet/machinelearning-modelbuilder#1562]
Liking the new updates to Model Builder.  Good job team 👍 

If I may ask, I noticed there is a link to samples for anomaly detection.  Do you have firm plans when we will see more support for related training and code generation for anomaly detection?  Thanks in advance.

**Comment by beccamc** [comment]
Thanks for the support, and the suggestion! 

@briacht Do we have an issue tracking anomaly detection?

**Comment by briacht** [comment]
I don't believe we do, but it is on the roadmap! I believe we will implement forecasting first and then anomaly detection.
